ood-portal-generator/templates/ood-portal.conf.erb

```diff
@@ -76,6 +76,22 @@ Listen <%= addr_port %>
   <%- end -%>
   <%- end -%>
 
+  <%- if @use_rewrites && @use_maintenance -%>
```

Comment:
Do we need a separate `@use_maintenance` config option or could we just have this always included? If the `/maintenance.html` file does not exist and rewrite engine is enabled this will just be ignored. I guess it does add one extra check for every request, but I don't know how expensive that is.

Reply:
By default `maintenance.html` would always exist if we removed `use_maintenance` check as that's also used in the `generate()` function of the main application.  This change would not alter a site's ood-portal.conf so there would be no Apache restart upon upgrade unless they explicitly enabled the maintenance rewrite rules.  We'd still need some way for people to turn this off if they don't want the extra rewrite rules.

Reply:
See below about enabling maintenance mode.

As for adding the rewrite rules, maybe we default it to true but keep @use_maintenance so the rewrites can be omitted if the site explicitly configures `use_maintenance: false`

Reply:
This change is done, now `use_maintenance` is on by default and can be turned off.